be/src/exec/vectorized/hdfs_scan_node.cpp

```diff
@@ -261,9 +261,6 @@ int HdfsScanNode::_compute_priority(int32_t num_submitted_tasks) {
 
 void HdfsScanNode::_scanner_thread(HdfsScanner* scanner) {
     Status status = scanner->open(_runtime_state);
-    if (!status.ok()) {
```

Comment:
We may encounter a logical empty file, but we can not set EOF right here. There are many other files be scanned.